Cooper/ClientApp/src.v2/app/layouts/game/game.layout.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-game',
```

Comment:
prefix app
Actually, for layout it is not necessary required to have a prefix at all

for layout it is better to add suffix layout: _prefix-name_-game-**layout** or game-layout